middleware-jackson/src/test/java/com/nytimes/android/external/store3/middleware/jackson/Model.kt

```diff
@@ -0,0 +1,9 @@
+package com.nytimes.android.external.store3.middleware.jackson
+
+data class Bar (val string: String)
+
+data class Foo (
+        val number: Int,
+        val string: String,
+        val bars: List<Bar>
+)
```

Comment:
Nit: new line feed at end of file! IntelliJ IDEA-based IDEs offer a setting for this as well. Search for `Ensure line feed at end of file`.